src/Cardano/AddressV3.cpp

```diff
@@ -20,7 +20,40 @@ using namespace TW;
 using namespace TW::Cardano;
 using namespace std;
 
-bool AddressV3::parseAndCheckV3(const std::string& addr, NetworkId& networkId, Kind& kind, Data& raw) {
+bool AddressV3::checkLength([[maybe_unused]] NetworkId networkId, Kind kind, size_t length) noexcept {
```

Comment:
Will `networkId` be used in the future?
```suggestion
bool AddressV3::validateLength(Kind kind, size_t length) noexcept {
```

Reply:
Thinking of it, highly unlikely that address content differs by network, plus we support Production net anyways.  Removing.